packages/kbn-i18n/README.md

```diff
@@ -252,32 +252,64 @@ and added as a comment next to translation message at `defaultMessages.json`
 
 
 #### Attributes translation in React
-React wrapper provides an API to inject the imperative formatting API into a React
-component by using render callback pattern. This should be used when your React
-component needs to format data to a string value where a React element is not
-suitable; e.g., a `title` or `aria` attribute. In order to use it, you should
-wrap your components into `I18nContext` component. The child of this component
-should be a function that takes `intl` object into parameters:
+
+React wrapper provides an ability to inject the imperative formatting API into a React component via its props using `injectI18n` HOC. This should be used when your React component needs to format data to a string value where a React element is not suitable; e.g., a `title` or `aria` attribute. In order to use it you should wrap your component into `injectI18n` function. The formatting API will be provided to the wrapped component via `props.intl`.
+
+React component as a pure function:
+
+```js
+import React from 'react';
+import { ReactI18n } from '@kbn/i18n';
```

Comment:
__note:__ don't forget to update this example in https://github.com/elastic/kibana/pull/20513 (or after it's merged, up to you).

Reply:
@azasypkin yes, believe it would be better to update this one after merging https://github.com/elastic/kibana/pull/20513